src/modules/fancyzones/editor/FancyZonesEditor/App.xaml.cs

```diff
@@ -13,18 +13,25 @@ namespace FancyZonesEditor
     /// </summary>
     public partial class App : Application
     {
-        public Settings ZoneSettings { get; }
+        public Settings[] ZoneSettings { get; }
+
+        public static int NumMonitors { get; private set; }
 
         public App()
         {
-            ZoneSettings = new Settings();
+            NumMonitors = Environment.GetCommandLineArgs().Length / 6;

```

Comment:
Can't the number of arguments change in the future? I wonder if there is another way to grab the number of monitors (we do this by parsing zone-settings.json and grab the devices array, like this the code would be resilient to changes in the number of arguments in the future)

Reply:
The number of arguments can change every time you click the "Edit Zones" button ~ it is resilient to changes. In any case, this code was *not* updated! Thank you for catching it (: